src/config/reader.rs

```diff
@@ -292,6 +299,17 @@ mod test_proto_config {
             .unwrap();
     }
 
+    #[tokio::test]
+    async fn test_proto_no_pkg() -> Result<()> {
+        let runtime = crate::runtime::test::init(None);
+        let reader = ConfigReader::init(runtime);
+        let mut proto_no_pkg = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        proto_no_pkg.push("tests/unit_tests/proto_no_pkg.graphql");
```

Comment:
Let's load it from `src/grpc/tests/proto` directory instead.

Reply:
will have to change existing tests then

Reply:
there is a test which reads all available proto files to check if the logic works for nested files and stuff